### PR TITLE
Updating VELPT filemasks

### PR DESCRIPTION
**Pull request by cdobs**
Corrected the recovered_inst VELPT filemasks for GS01SUMO R00004 and GA01SUMO R00003. Also GI01SUMO R00005 ingestion csv which was missing and included correct velpt filemask.

### CODE REVIEW

**Comment by cdobs** [comment]
[Re-ingest Global VELPT data [DT-103]](https://trello.com/c/WHz5Z7z5/103-re-ingest-global-velpt-data-dt-103)

File: GA01SUMO/GA01SUMO_R00003_ingest.csv [hunk 1]
```diff
@@ -5,7 +5,7 @@ mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_recovered_driver,/omc_data/whoi/OMC/
 mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/GA01SUMO/R00003/cg_data/dcl16/ctdbp/*ctdbp.log,GA01SUMO-RID16-03-CTDBPF000,recovered_host,Available,
 mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/GA01SUMO/R00003/instruments/dcl16/CTDBP_sn16-50062/*.hex,GA01SUMO-RID16-03-CTDBPF000,recovered_inst,Available,Only partial (~1.4%) recovered data from instrument due to damaged bulkhead
 mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_recovered_driver,/omc_data/whoi/OMC/GA01SUMO/R00003/cg_data/dcl16/velpt/*.velpt.log,GA01SUMO-RID16-04-VELPTA000,recovered_host,Available,
-mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/GA01SUMO/R00003/instruments/dcl16/VELPT_snAQD12229/1222901.aqd,GA01SUMO-RID16-04-VELPTA000,recovered_inst,Available,Not parsed on local machine
+mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/GA01SUMO/R00003/instruments/dcl16/VELPT_sn*/*.aqd,GA01SUMO-RID16-04-VELPTA000,recovered_inst,Available,
```
Comment (snwhite) [inline GA01SUMO/GA01SUMO_R00003_ingest.csv:8]: Looks good.

File: GS01SUMO/GS01SUMO_R00004_ingest.csv [hunk 2]
```diff
@@ -5,7 +5,7 @@ mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_recovered_driver,/omc_data/whoi/OMC/
 mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl16/ctdbp/*ctdbp.log,GS01SUMO-RID16-03-CTDBPF000,recovered_host,Available,
 mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl16/CTDBP_sn16-*/*.hex,GS01SUMO-RID16-03-CTDBPF000,recovered_inst,Available,
 mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl16/velpt/*.velpt.log,GS01SUMO-RID16-04-VELPTA000,recovered_host,Available,
-mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl16/VELPT_sn*/*.dat,GS01SUMO-RID16-04-VELPTA000,recovered_inst,Available,
+mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl16/VELPT_sn*/*.aqd,GS01SUMO-RID16-04-VELPTA000,recovered_inst,Available,
```
Comment (snwhite) [inline GS01SUMO/GS01SUMO_R00004_ingest.csv:8]: Looks good.

File: GI01SUMO/GI01SUMO_R00005_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00005/cg_data/dcl12/hyd2/*hyd2.log,GI01SUMO-SBD12-03-HYDGN0000,recovered_host,,
+mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00005/cg_data/dcl12/pco2a/*.pco2a.log,GI01SUMO-SBD12-04-PCO2AA000,recovered_host,,
+mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00005/cg_data/dcl12/wavss/*.wavss.log,GI01SUMO-SBD12-05-WAVSSA000,recovered_host,,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00005/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA000,recovered_host,,
+mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00005/cg_data/dcl12/fdchp/*.fdchp.log,GI01SUMO-SBD12-08-FDCHPA000,recovered_host,,
+#mi.dataset.driver.fdchp_a.fdchp_a_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00005/instruments/dcl12/FDCHP_sn*/D*/fdchp_*.dat,GI01SUMO-SBD12-08-FDCHPA000,recovered_inst,Not available,Instrument failed on deployment
```
Comment (snwhite) [inline GI01SUMO/GI01SUMO_R00005_ingest.csv:76]: Looks good.

**Pull request merged**